install-deps.sh

```diff
@@ -1,38 +1,29 @@
-#!/bin/bash -e
-#
 # Ceph distributed storage system
 #
 # Copyright (C) 2014, 2015 Red Hat <contact@redhat.com>
 #
 # Author: Loic Dachary <loic@dachary.org>
+# Modified: Shinobu Kinjo <skinjo@redhat.com>
 #
 #  This library is free software; you can redistribute it and/or
 #  modify it under the terms of the GNU Lesser General Public
 #  License as published by the Free Software Foundation; either
 #  version 2.1 of the License, or (at your option) any later version.
 #
+LSBR=`which lsb_release`
+DIST=`$LSBR -si`
```

Comment:
Yes, that's my mistake. Those lines come after checking /etc/issue and installing lsb_releaes.